.github/workflows/set_environment_variables.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+os=$1
+build_type=$2
+arch=$3
+workspace=$4
+
+
+if [[ "$os" == "" || "$build_type" == "" || "$arch" == "" || "$workspace" == "" ]]; then
+	echo "Invalid parameters"
+	exit 1
+fi
+
+# Turn variables into lowercase
+os="${os,,}"
+# only consider name up to the hyphen
+os=$(echo "$os" | sed 's/-.*//')
+build_type="${build_type,,}"
+arch="${arch,,}"
+
+
+MUMBLE_ENVIRONMENT_DIR="$workspace/mumble-build-environment"
+MUMBLE_BUILD_ENV_PATH=""
+MUMBLE_ENVIRONMENT_VERSION=""
+ADDITIONAL_CMAKE_OPTIONS=""
+VCPKG_CMAKE_OPTIONS=""
+
+case "$os" in
+	"ubuntu")
+		# We have to use the version without debug symbols in order for the size of the
+		# uncompressed archive to not exceed CI size limits
```

Comment:
Does this apply to GitHub Actions as well?

Reply:
Yes - I tested it ^^
The very curious thing is that it works for the ubuntu-20.04 image but not for ubuntu-18.04. The latter has slightly less disk space available for some reason :shrug:

Reply:
Maybe the Ubuntu 18.04 image actually weighs more? Or maybe a larger virtual disk is used to accommodate for the larger image.